internal/cli/logout.go

```diff
@@ -21,7 +21,7 @@ func logoutCmd(cli *cli) *cobra.Command {
 			if len(args) == 0 {
 				tens, err := cli.listTenants()
 				if err != nil {
-					return fmt.Errorf("Unable to load tenants due to an unexpected error: %w", err)
+					return err // This error is already formatted for display
```

Comment:
Is it possible for another err other than `errUnauthenticated` to be returned? Do we need to check that it is that error?

Reply:
Good Q @aherzog-auth0 . It boils down to two other error types:

- json.Unmarshal
- os.ReadFile

If we want this to be complete, we can check `errUnauthenticated`, return it as is, otherwise do an error wrap with `Unexpected error: %w` type pattern.

What do you think @Widcket ?